src/pages/upload/UploadPage.js

```diff
@@ -0,0 +1,144 @@
+    };
+
+    return (
+        <div className="upload">
+            <div className="container">
+                <Row>
+                    <Col span={12}>
+                        <div className="form-upload">
+                            <Form
+                                layout="vertical"
+                                form={form}
+                                requiredMark={false}
+                                onFinish={onFinish}
+                            >
+                                <p className="title-item-upload">Item Upload</p>
+                                <Form.Item
+                                    name="productName"
+                                    rules={[
+                                        {
+                                            required: true,
+                                            message: "* Please input name product",
+                                        },
+                                    ]}
+                                >
+                                    <p className="label">Product Name *</p>
+                                    <Input />
+                                </Form.Item>
+
+                                <Form.Item
+                                    name="price"
+                                    rules={[
+                                        {
+                                            required: true,
+                                            message: '* Price must required',
+                                        },
+                                    ]}
+                                >
+                                    <p className="label">Price *</p>
+                                    <Input />
+                                </Form.Item>
+
+                                <p className="title-item-details">Item Details</p>
+                                <Form.Item
+                                    name="itemDetails"
+                                    rules={[
+                                        {
+                                            required: true,
+                                            message: '* Select Category',
+                                        },
+                                    ]}
+                                >
+                                    <p className="label">Category *</p>
+                                    <Select defaultValue="Clothes">
+                                        <Option value="Clothes">Clothes</Option>
+                                        <Option value="shoes">Shoes</Option>
+                                        <Option value="pants">pants</Option>
+                                    </Select>
+                                </Form.Item>
+
+                                <Form.Item style={{ marginBottom: 0 }}>
+                                    <p className="label">Varian</p>
+                                    <Form.Item
```

Comment:
Use this component instead: https://ant.design/components/select/#components-select-demo-tags